src/core/xfa/xhtml.js

```diff
@@ -109,6 +111,7 @@ const StyleMapping = new Map([
 
 const spacesRegExp = /\s+/g;
 const crlfRegExp = /[\r\n]+/g;
+const crlfForRichTextRegExp = /\r\n|\r/g;
```

Comment:
Nit: Unless I'm missing something, I *think* that you could just use `/\r\n?/g` instead.